my-app-react/src/Components/CartItems/CartItems.js

```diff
@@ -1,41 +1,69 @@
 import "./cartItems.style.css";
 import { VegLogo, NonVegLogo } from "../../Models/ImageConstants";
-function CartItems(props) {
-  console.log("Re render :", props.allCartItems.length);
-  function increaseTheCount(dishName) {
-    props.increaseInCart(dishName);
-  }
-  function decreaseTheCount(dishName) {
-    props.decreaseInCart(dishName);
-  }
+import { connect } from "react-redux";
+import { getCartListByName } from "../../redux/selectors";
+import {
+  increaseToCart,
+  decreaseToCart,
+  removeFromCart,
+} from "../../redux/actions";
+function CartItems({
+  cartItems,
+  increaseToCart,
+  decreaseToCart,
+  removeFromCart,
+}) {
   let totalqty = 0;
```

Comment:
no let or for loop, use const and forEach, map or reduce based on the requirement.

Reply:
done